src/extensions/tests.rs

```diff
@@ -0,0 +1,136 @@
+use crate::{gtk, RelmIterChildrenExt, RelmListBoxExt, RelmRemoveAllExt};
+use gtk::prelude::{BoxExt, GridExt, WidgetExt};
+
+// A set of widgets for tests
+#[derive(Default)]
+struct TestWidgets(gtk::Label, gtk::Switch, gtk::Box);
+
+impl TestWidgets {
+    fn assert_parent(&self) {
+        assert!(self.0.parent().is_some());
+        assert!(self.1.parent().is_some());
+        assert!(self.2.parent().is_some());
+    }
+}
+
+// Returns whether two optional widgets are the same.
+fn same_widgets(
+    first: Option<impl AsRef<gtk::Widget>>,
+    second: Option<impl AsRef<gtk::Widget>>,
+) -> bool {
+    match (first, second) {
+        (Some(first), Some(second)) => first.as_ref() == second.as_ref(),
+        (None, None) => true,
+        _ => false,
+    }
+}
+
+#[gtk::test]
+fn box_ext() {
```

Comment:
A name like `box_extension_trait_test` would be more fitting now that `BoxExt` was removed. Same for grid and flow_box.

Reply:
`box_extension_traits` should be fine.